src/utils/flow-manager.ts

```
@@ -5,40 +5,126 @@ import { injectable, inject } from 'tsyringe';
+        const refStepId = ValidationUtils.validateStringField(
+          value,
+          key,
+          'NextStepId'
+        );
+        if (!stepIds.has(refStepId)) {
+          throw new Error(`Invalid nextStepId reference: ${refStepId}`);
+        }
+      }
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)